packages/contract-artifacts/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": {
+        "outDir": "lib",
+        "rootDir": ".",
+        "resolveJsonModule": true
+    },
+    "include": ["./globals.d.ts", "./src/**/*"],
```

Comment:
Where is the `globals.d.ts`?

Reply:
Just removed this. `global.d.ts` was required before I realized we could use `resolveJsonModules` here.